xw2600.md

```diff
@@ -0,0 +1,35 @@
+# Open Source Development
+Link to my open source project repo: https://github.com/rennahweng/two-bias/tree/master
```

Comment:
shorten to just https://github.com/rennahweng/two-bias

Reply:
Okie, changed!